### ISSUE TITLE
ERROR: AJAX Loading Error: Forbidden

### ISSUE THREAD

**Issue by WebVantage** [joomla/joomla-cms#11635]
### Steps to reproduce the issue
Attempted to install via Live Update and Upload & Update and got the same result "ERROR: AJAX Loading Error: Forbidden"

### Expected result
Update from Joomla 3.6.1 to 3.6.2

### Actual result
"ERROR: AJAX Loading Error: Forbidden"

### System information (as much as possible)
PHP Built On	Linux host2.yourwebvantage.com 2.6.32-573.12.1.el6.x86_64 #1 SMP Tue Dec 15 21:19:08 UTC 2015 x86_64
Database Version	5.5.50-cll
Database Collation	latin1_swedish_ci
Database Connection Collation	utf8mb4_general_ci
PHP Version	5.6.21
Web Server	Apache
WebServer to PHP Interface	cgi-fcgi
Joomla! Version	Joomla! 3.6.1 Stable [ Noether ] 2-August-2016 22:59 GMT
Joomla! Platform Version	Joomla Platform 13.1.0 Stable [ Curiosity ] 24-Apr-2013 00:00 GMT
User Agent	Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/601.7.7 (KHTML, like Gecko) Version/9.1.2 Safari/601.7.7

**Comment by C-Lodder** [comment]
As suggested by @nikosdion:

> The Forbidden error message means that your server denied serving the request to the update extraction script. I would recommend doing the following:
> - Rename your .htaccess file to .htaccess.bak
> - Run the update
> - Rename .htaccess.bak back to .htaccess

I assume you already Googled the error?

**Comment by brianteeman** [comment]
I am closing this at this time as not a core joomla bug. It can always be reopened if shown to be otherwise

**Comment by CSchaefer70** [comment]
I had this problem with Joomla 3.48 attempting to upload to 3.6.5... C-Lodder suggestions (di step by step) worked for me without and issues... Thank you so much!

**Comment by davidsbach** [comment]
I have had this error attempting to use Joomla update for a long time. I have tried renaming the htaccess file land I still get the Ajax error.
The site is just like two other Joomla! sites I maintain and Joomla update works fine on them. I have looked at permissions and don't see anything suspicious.
Previously, FTPd the update and then used postupdate.php successfully. I read now that it no longer works. 
What are my options?
